src/pages/product/documentation.js

```diff
@@ -0,0 +1,181 @@
+      ],
+    },
+    {
+      title: 'Centralized yet distributed Docs-as-Code',
+      description: 'Roadie lets developers maintain their docs along their codebase',
+      illustration: {
+        png: DocsMarkDown,
+        alt: '',
+      },
+      paragraphs: [
+        'Outdated documentation is often counterproductive. Roadie’s Tech Docs are markdown files that to live next to the code they document, so developers have it easier to update it. Roadie will go through your repos looking for documentation entries and bring them all into your Internal Developer Portal.',
+      ],
+    },
+    {
+      title: 'A home for all API Specs too',
+      description: 'Roadie lets you bring in OpenAPI, AsyncAPI, GraphQL, gRPC and custom API docs',
+      illustration: {
+        png: DocsApitypes,
+        alt: '',
+      },
+      paragraphs: [
+        'Minimize context switching by letting devs discovering an API in the catalog and learning how to use it on the spot. When your teams register API into the Roadie Catalog, they can let it render their endpoints documentation right within the corresponding entity page. Roadie will also have an aggregated view page for developers to have a single reference for all API docs.',
+      ],
+    },
+    {
+      title: 'Avoid proprietary lock-ins',
+      description: 'Roadie gives you the freedom of Open Source',
+      illustration: {
+        png: NoLockInImg,
+        alt: '',
+      },
+      paragraphs: [
+        'Spotify open-sourced Backstage because they didn’t want to re-do all their Developer Portal if a popular vendor came around. By using Roadie, you’re adopting Open Source Backstage but without the extensive setup and heavy maintenance.',
+      ],
+    },
+  ],
+};
+
+const Home = ({ data }) => {
+  const siteTitle = data.site.siteMetadata.title;
+  const darkLogos = [...LOGOS].filter(({ src }) => has(src, 'dark')).slice(0, 5);
+
+  return (
+    <>
+      <SEO title={`${SEO_TITLE} | ${siteTitle}`} description={LEAD} />
+
+      <SitewideHeader borderBottom={false} />
+
+      <section className="bg-blueroadie mx-auto max-w-7xl xl:rounded-lg lg:flex items-center">
+        <div className="lg:w-1/2 p-4 lg:px-10 lg:py-16">
+          <strong className="block text-white uppercase mb-8 text-xl font-highlight">
+            Roadie’s Documentation
+          </strong>
+          <Headline size="medium" className="text-white">
+            <span className="text-orange-600">Centralized docs:</span> no more shuffling around for
+            documentation
+          </Headline>
+
+          <h2 className="mt-5 text-white text-lg sm:mt-8 lg:text-xl xl:text-xl xl:mr-6">
+            Roadie’s Tech Docs lets your developers write their docs in their own repo, and find
```

Comment:
This sentence reads weird to me. How about "Roadie’s Tech Docs lets developers write docs alongside code, but read them in their Internal Developer Portal."